src/components/company-list-item/company-list-item.js

```diff
@@ -31,7 +31,7 @@ export default class CompaniesListItem extends Component {
 
 
     render() {
-        const { name, address, phonesNumber, site, description, onDelete, onEdit} = this.props
+        const { name, address, phonesNumber, site, description, onDelete, onEdit } = this.props
```

Comment:
instead of each company property as props(name, address...) use company object. So you will have:
```
  const { company, onDelete, onEdit } = this.props;
```

Reply:
`;` after each javascript expression

Reply:
done